bento/src/main/java/com/yelp/android/bento/core/SmartAsyncInflationCache.kt

```diff
@@ -0,0 +1,31 @@
+package com.yelp.android.bento.core
+
+private const val CACHE_SIZE_LIMIT = 30
```

Comment:
There seems to be a mechanism that can be used by LinkedHashMap to limit the size to x. In this exemple: https://stackoverflow.com/a/11469731/870751 they implement the removeEldestEntry and it seems to automatically remove the oldest one past the size limit.
It would be useful here!

Reply:
That's great! Thanks.